tests/test_cli.py

```diff
@@ -140,6 +140,86 @@ def test_create_user_config() -> None:
     }
 
 
+@pytest.mark.parametrize(
+    ("user_config_args,config_type,str_value"),
+    [
+        ('{"description":"test","network":"0.0.0.0/0"}', "array", [{"description": "test", "network": "0.0.0.0/0"}]),
```

Comment:
If a single JSON object as a string is parsed into a one element array, then what kind of input is needed for a multi-element one (i.e. multiple IP filters with descriptions)? It's not immediately obvious to me when reading the above code, so I'd like to at least have a test case for that scenario.

Reply:
It would fail. I think we can have a test case for that